docs/development.rst

```diff
@@ -203,6 +191,42 @@ Then execute the following and copy the markdown version for pasting to GitHub r
 
 After the release commit has been pushed and a release has been tagged, set a development version in the same above files. This is basically the next minor release postfixed by ``-dev``.
 
+Developing with Docker
+------------------------
+
+If you choose, you may develop Socialhome using Docker, rather than installing Postfix and Redis manually on your computer.
```

Comment:
`Postfix` -> `PostgreSQL`.

Maybe someday we *will* have a postfix dependency to allow Socialhome to act as a mail server ;)

Reply:
Yes, this is actually a thing to think about in production (since notifications are via mail) but it's fixed in my current version. :heavy_check_mark: